samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -14,53 +14,60 @@
 
 namespace FunctionApp
 {
-    public class SimpleChat : ServerlessHub
+    public class SimpleChat : ServerlessHub<SimpleChat.IChatClient>
     {
         private const string NewMessageTarget = "newMessage";
         private const string NewConnectionTarget = "newConnection";
 
+        public interface IChatClient
+        {
+            public Task newConnection(NewConnection newConnection);
+            public Task newMessage(NewMessage newMessage);
+        }
+
         [FunctionName("negotiate")]
-        public Task<SignalRConnectionInfo> NegotiateAsync([HttpTrigger(AuthorizationLevel.Anonymous)] HttpRequest req)
+        public async Task<SignalRConnectionInfo> NegotiateAsync([HttpTrigger(AuthorizationLevel.Anonymous)] HttpRequest req)
         {
             var claims = GetClaims(req.Headers["Authorization"]);
-            return NegotiateAsync(new NegotiationOptions
+            var result = await  NegotiateAsync(new NegotiationOptions
```

Comment:
why change this?

Reply:
The `NegotiateAsync` method returns `ValueTask` for strongly-typed hub.